src/pages/heeyeolKim/Main/Main.js

```diff
@@ -0,0 +1,318 @@
+                                        type="text" 
+                                        placeholder="댓글달기..."
+                                        value={this.state.newComment}
+                                        onChange = {this.createNewComment}
+                                        />
+                                    </div>
+                                    <div>
+                                        <button 
+                                        className="uploadComment"
+                                        onClick = {this.uploadNewComment}
+                                        >
+                                            <a href="#!">
+                                                <span>게시</span>
+                                            </a>
+                                        </button>
+                                    </div>
+                                </div>
+                            </section>
+                        </article>
+                    </main>
+
+                    <aside>
+                        <div className="asideContainer">
+                            <div className="asideProfile">
+                                <div className="asideProfileContainer">
+                                    <div className="asideProfileImage">
+                                        <img src={story1} alt=""/>
+                                    </div>
+                                    <div className="asideProfilename">
+                                        <ul>
+                                            <li>hereizhere</li>
+                                            <li>김희열</li>
+                                        </ul>
+                                    </div>
+                                </div>
+                                <div className="asideProfileChange">
+                                    전환
+                                </div>
+                            </div>
+                            <ul className="asideRecomment">
+                                <li className="asideRecommentForMe">회원님을 위한 추천</li>
+                                <li className="asideRecommentAll">모두 보기</li>
+                            </ul>
+                            <ul className="recommentList">
+                                <ul className="recommentListLeft">
+                                    <li className="recommentListImage">
+                                        <img src={p1} alt=""/>
+                                    </li>
+                                    <li>
+                                        <ul className="recommentListCenter">
+                                            <li>monday</li>
+                                            <li>회원님을 팔로우합니다.</li>
+                                        </ul>
+                                    </li>
+                                </ul>
```

Comment:
```suggestion
                                <div className="recommentListLeft">
                                    <li className="recommentListImage">
                                        <img src={p1} alt=""/>
                                    </li>
                                    <li>
                                        <ul className="recommentListCenter">
                                            <li>monday</li>
                                            <li>회원님을 팔로우합니다.</li>
                                        </ul>
                                    </li>
                                </div>
```
나중에 추가되면 모르겠지만 지금은 ul보단 div가 더 적절해 보입니다!

Reply:
- li 태그는 항상 ul 태그 안에 위치하게 해주시고,
- 지금 li 태그가 둘로 분리되어 있는데 (근데 li 태그 안에 ul 태그가 또 있네요? 그 안에 또 li 태그가 있고..)
- ul 태그 안에 li 태그, li 태그 안에 img, span, div 와 같은 태그 써서 하나의 단위를 li 태그로 잘 묶어주세요.

Reply:
수정 완료 했습니다. 🙌